src/com/lftechnology/java/training/dipak/employeemanagement/ui/DuplicateEmployee.java

```diff
@@ -0,0 +1,26 @@
+
+package com.lftechnology.java.training.dipak.employeemanagement.ui;
+
+import com.lftechnology.java.training.dipak.employeemanagement.domain.Employee;
+
+/**
+ * <p>This class contains a static method that copies the Employee object to another Employee object.</p>
+ * 
+ * @author Dipak Thapa<dipakthapa@lftechnology.com>
+ */
+public class DuplicateEmployee {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a private constructor to hide the implicit public one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1118)